lib/src/domain/web_rtc/web_rtc_client/web_rtc_peer_connection.dart

```diff
@@ -31,15 +31,29 @@ class WebRtcPeerConnection {
   }
 
   Future<void> _webRTCInit() async {
+    final config = await _webRtcDirectDataSource.getOptionalWebRtcConfig();
+    final additionalIceServers = config.additionalIceServers.map((e) => {
+          'urls': e.urls,
+          'credential': e.credential,
+          'username': e.username,
+        });
+
     ///create Peer;
-    final rtcConfig = {
-      'iceServers': [
-        {
-          "urls": "stun:global.stun.twilio.com:3478?transport=udp",
-          'sdpSemantics': 'unified-plan',
-        },
-      ]
-    };
+    final Map<String, List<dynamic>> rtcConfig;
+    if (additionalIceServers.isEmpty) {
+      rtcConfig = {
+        'iceServers': [
+          {
+            'urls': 'stun:global.stun.twilio.com:3478?transport=udp',
+            'sdpSemantics': 'unified-plan',
+          },
+        ]
+      };
+    } else {
+      rtcConfig = {
+        'iceServers': additionalIceServers.toList(),
+      };
+    }
```

Comment:
we should include the default stun server even if additional ice servers are available